src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from "react"
+import { getArticleById } from "../api"
+
+const ArticleCard = (props) => {
+    const { article } = props
+    const [articleCard, setArticleCard] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    const [isError, setIsError] = useState(false)
+    
+    useEffect(() => {
+        setIsLoading(true)
+        getArticleById(article.article_id)
```

Comment:
So at the moment,  you're getting all of your articles up in ArticlesList, but then other than mapping through the array, you're not doing anything with that data, other than triggering another request for each article down in article card.